src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java

```diff
@@ -2,7 +2,9 @@
 
 import br.com.caelum.clines.api.airports.ExistingLocationService;
 import br.com.caelum.clines.shared.domain.Location;
+import br.com.caelum.clines.shared.infra.Mapper;
 import org.springframework.data.repository.Repository;
 
 public interface LocationRepository extends Repository<Location, Long>, ExistingLocationService {
+    void save(Location location);
```

Comment:
Spring interface `Repository` already has a method `save(E entity)`. I don't think it is necessary redeclarate it.

Reply:
`save(E entity)` is only available in `CrudRepository` or `PaggingAndSortRepository`

Reply:
My mistake then.